src/types/index.ts

```diff
@@ -118,8 +118,9 @@ export interface IPageableResponse<T> {
 }
 
 export interface IAppData {
-  name: string;
-  url: string;
+  href?: string;
```

Comment:
What is the difference between `href` and `url`? If both are to store full url I suggest to use one option.